Makefile

```diff
@@ -286,12 +286,16 @@ test-integration:
 #############
 
 package-wapm:
-ifneq ($(OS), Windows_NT)
 	mkdir -p "package/bin"
+ifneq ($(OS), Windows_NT)
+	if [ -d "wapm-cli" ]; then \
+		cp wapm-cli/target/release/wapm package/bin/ ;\
```

Comment:
I think we need to move this up too

Reply:
Oh, got confused!

Reply:
I tried to but I couldn't get the `ifneq` working internally because the outer loop was bash, so I settled to just duplicate it!